### PR TITLE
Check for no internet

### PR DESCRIPTION
**Pull request by steam**
In order to create a more "app-like" experience we want to show a native Android screen when the user is not connected to the internet. This prevents the less than ideal scenario of the user attempting to load the website in the shell and seeing a browser error.

screen shot 2016-03-11 at 8 38 56 am

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
         android:label="@string/app_name"
-        android:theme="@style/AppTheme" >
+        android:theme="@style/AppTheme">
         <activity
             android:name=".MainActivity"
+            android:configChanges="keyboardHidden|orientation|screenSize"
             android:label="@string/app_name"
             android:screenOrientation="portrait"
-            android:configChanges="keyboardHidden|orientation|screenSize"
+            android:launchMode="singleTop"
```
Comment (steam) [inline app/src/main/AndroidManifest.xml]: This ensures that there is only ever one Ello `MainActivity` running on the device.

**Comment by steam** [comment]
Any chance I can get a set of eyes on this @colinta?

File: app/src/main/java/ello/co/ello/ElloUtil.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+import android.app.AlertDialog;
+import android.content.Context;
+import android.net.ConnectivityManager;
+import android.net.NetworkInfo;
+
+/**
+ * Created by s on 3/4/16.
+ */
+public class ElloUtil {
```
Comment (colinta) [inline app/src/main/java/ello/co/ello/ElloUtil.java]: This starts down a dangerous path of "`ElloUtil` dumping ground".  `isNetworkConnected -> Bool` and `showErrorNoInternet` are very different beasts (internal state check vs user interaction).  I would rather see `Reachability.isNetworkConnected()` and `Alert.showErrorNoInternet(context)`.  (names can be anything, I used these just as examples)

**Comment by colinta** [comment]
Other than comment about `ElloUtil`, looks great!

File: app/src/main/java/ello/co/ello/ElloUtil.java (hunk 2 above)
Comment (steam) [inline app/src/main/java/ello/co/ello/ElloUtil.java]: I agree. These were essentially ported over from some code that Justin had in another project and I was sloppy. I'll split them up. Thanks :metal:

**Pull request merged**